docs/x509/ocsp.rst

```diff
@@ -285,6 +286,55 @@ Creating Responses
             :class:`~cryptography.x509.ReasonFlags` enumeration or ``None`` if
             the ``cert`` is not revoked.
 
+    .. method:: add_response_by_hash(issuer_name_hash, issuer_key_hash, serial_number, algorithm, cert_status, this_update, next_update, revocation_time, revocation_reason)
+
+        .. versionadded:: 45.0.0
+
+        This method adds status information about the certificate that was
```

Comment:
I think we should add an example that shows correct generation of the hashes for this.

Reply:
We don't have that for the OCSPRequest side, fwiw

Reply:
We can always be better 😇

Reply:
shiiit

Reply:
Because we lack an API for getting the SPKI value bytes (i.e., excluding the OID), it's not particularly clear how to write an example for this.